requirements/requirements_nlp.txt

```diff
@@ -5,4 +5,9 @@ sentencepiece
 torchtext
 transformers
 unidecode
-youtokentome
\ No newline at end of file
+youtokentome
+numpy
+tqdm
+sklearn
+requests
```

Comment:
I did not find requests, wget in nemo_nlp collection. could you delete these, bash scripts should not count in my opinion since they are not part of the package

Reply:
you're right, I just though we had them in the downloading .py scripts. Removed